react/__tests__/Telemarketing.test.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { render } from 'react-testing-library'
+
+import Telemarketing from '../components/Telemarketing'
+import messages from '../../messages/en-US.json'
+
+describe('<Telemarketing /> component', () => {
+  const intl = {
+    formatMessage: ({ id = '' }) => messages[id],
+  }
+
```

Comment:
What do you think to test if the component renders?

Reply:
TBH, I tested using the `expect(fragment).toBeTruthy()` with a component that returns null or undefined and it passes in the test. Seems like `react-testing-library` always add the `DocumentFragment` tag so it will be always Truthy.